settings.py

```diff
@@ -0,0 +1,12 @@
+from dotenv import load_dotenv
+from pathlib import Path
+import os
+
+load_dotenv()
+ENV_PATH = Path(__file__)/".env"
+load_dotenv(dotenv_path=ENV_PATH)
+
+NEO4J_USER = os.getenv("NEO4J_USER")
```

Comment:
Good that you didn't post the secrets to github

Reply:
I did it before and I will make sure always never happen to me again